tests/test_reader.py

```diff
@@ -695,12 +696,15 @@ def test_issue604(caplog, strict):
             ]
             assert normalize_warnings(caplog.text) == msg
 
-        def get_dest_pages(x) -> int:
+        def get_dest_pages(x) -> Union[int, None]:
             if isinstance(x, list):
                 r = [get_dest_pages(y) for y in x]
                 return r
             else:
-                return pdf.get_destination_page_number(x) + 1
+                des_page_number = pdf.get_destination_page_number(x)
```

Comment:
Could we please use a consistent naming pattern for "destination"? `dest_page_number` or `destination_page_number` appear to be more appropriate here.